pandas/core/arrays/integer.py

```diff
@@ -195,6 +198,23 @@ def coerce_to_array(values, dtype, mask=None, copy=False):
     return values, mask
 
 
+def _numpy_to_bitarray(arr):
```

Comment:
This and the method below maybe belong in a shared module, but placing here now for better change visibility